internal/fbsemantic/semantic.fbs

```diff
@@ -4,6 +4,15 @@ table Fresher {
   u:ulong;
 }
 
+table Packages {
+  packages:[PackageExports];
+}
+
+table PackageExports {
+  id:string;
+  package:TypeEnvironment;
+}
```

Comment:
I like these changes to make explicit types for exports from a package and a set of packages. They make the intent of the code clear.